packages/orchestrator-ui-components/src/components/WfoTree/treeUtils.ts

```diff
@@ -32,3 +41,38 @@ export const sortTreeBlockByLabel = (
     }
     return 0;
 };
+
+const countChildeNodes = (node: TreeBlock): number => {
```

Comment:
`getChildrenNodesCount` or `getNumberOfChildrenNodes`

Reply:
changed it to the little more accurate countDescendantNodes. Count is still a verb so I think it works